### ISSUE TITLE
Allow truthy/falsy index?

### ISSUE THREAD

**Issue by ZaxR** [jnothman/UpSetPlot#74]
I have a pandas dataframe with 1 or 0 as valued for dummied columns. When I try to plot without changing the 1/0 to True/False the figure is missing the dot grid/connections. Is this by design, and is it possible to allow the 1/0?

Below is an example to reproduce. The line with the comment is the one to include/remove to see the difference. Including images of the output for each below as well.

```
import pandas as pd
from matplotlib import pyplot as plt
from upsetplot import plot

df = pd.DataFrame({"unique_id": [1, 2, 3, 4, 5, 6, 7], "a": [1, 1, 1, 0, 1, 0, 0], "b": [1, 0, 0, 0, 1, 0, 0], "c": [1, 0, 1, 0, 1, 0, 0]})
df = df.replace({0: False, 1: True})  # different results with/without this conversion
plot_df = df.fillna(False).groupby(["a", "b", "c"]).count()['unique_id']
plot(plot_df)
pyplot.show()
```

Without the conversion to True/False:
![Screen Shot 2019-08-13 at 6 49 26 PM](https://user-images.githubusercontent.com/19669890/62985126-7a6e4a00-bdfb-11e9-9121-57026e924966.png)

With the conversion to True/False:
![Screen Shot 2019-08-13 at 6 49 38 PM](https://user-images.githubusercontent.com/19669890/62985121-76422c80-bdfb-11e9-90a2-e07c73a4e2f7.png)

**Comment by jnothman** [comment]
I think it should currently be raising an error with a non-boolean index
but it seems this is not the case.

> Do you think, if it did a better job of protecting against misuse, we
really need to support a truthy index?

Also, since I'm trying to improve the data format support and
documentation, is there a reason you find this easier than using the
from_memberships or from_contents helpers?

**Comment by ZaxR** [comment]
Thanks for the quick reply! I'm usually starting from a pandas dataframe that's the result of pd.get_dummies. The default dtype is np.uint8. As of pandas 0.23.0 you can set the dtype manually to bool, but then I'd need a whole temporary copy of my dataset. I just started experimenting with upsetplot, so open to a different creation method, if there's an easier way from my starting point.

**Comment by jnothman** [comment]
Oooh. I'd not seen `pandas.Series.str.get_dummies`... the ordinary `get_dummies` doesn't support multilabel, does it?

I've got a fix proposed in #76. Feel free to add a test for me! I don't have a lot of time to work on this.
